src/VisualStudio/Core/Def/Implementation/VisualStudioSupportsFeatureService.cs

```diff
@@ -31,13 +33,24 @@ public VisualStudioTextBufferSupportsFeatureService()
             }
 
             public bool SupportsCodeFixes(ITextBuffer textBuffer)
-                => SupportsCodeFixesWorker(GetContainedDocumentId(textBuffer));
+            {
+                return SupportsCodeFixesWorker(GetContainedDocumentId(textBuffer));
+            }
 
             public bool SupportsRefactorings(ITextBuffer textBuffer)
-                => SupportsRefactoringsWorker(GetContainedDocumentId(textBuffer));
+            {
+                return SupportsRefactoringsWorker(GetContainedDocumentId(textBuffer));
+            }
 
             public bool SupportsRename(ITextBuffer textBuffer)
             {
+                // Disable rename entry points when running under the LSP editor.
+                // The LSP client will handle the rename request.
+                if (textBuffer.IsInLspEditorContext())
```

Comment:
Similar question as earlier: should this be elsewhere?

Reply:
🤦